decidim-accountability/lib/decidim/accountability/version.rb

```diff
@@ -4,7 +4,7 @@ module Decidim
   # This holds decidim-accountability version.
   module Accountability
     def self.version
-      "0.7.0"
+      "0.7.1"
```

Comment:
Don't quite understand why we can't get this version from `Decidim::Core`, as it's a hard dependency. But ok!

Reply:
Reasoning is in https://github.com/decidim/decidim/pull/2031, which was merged 2 weeks ago.

Reply:
Yeah, that took care of `require_relative` issues, but in this scenario we could be easily using `require` as `decidim-core` is a hard dependency. I'll try to look at it further, since I think we can improve this.

Reply:
@josepjaume I tried your suggestion at the time, but there was a deeper problem: version files are evaluated very early (when gemspecs are read, to set the gem's version). My experience was that requiring dependencies at that point doesn't reliably work, since the `LOAD_PATH` is not yet fully available.

So I went with the `Rails` & `RSpec` solution, that also gives us full flexibility. For example, `RSpec` sometimes releases a new patch-level version of, say `rspec-support` without having to release the other gems.

In terms of maintainability, these files only ever change when version changes, and we have a `rake` task to manage that, so it doesn't seem like a big deal.

What do you think?

Reply:
Oh, right @deivid-rodriguez! Now I get it. Thanks for the explanation!